### PR TITLE
Fix postgres preflight condition [v1.3]

### PR DESCRIPTION
**Pull request by erzetpe**

### CODE REVIEW

**Review by seriva — DISMISSED** [review]

**Review by cicharka — DISMISSED** [review]

File: docs/changelogs/CHANGELOG-1.3.md [hunk 1]
```diff
@@ -45,6 +45,7 @@
 - [#2420](https://github.com/epiphany-platform/epiphany/issues/2420) - Changing Grafana admin password in the apply mode
 - [#2873](https://github.com/epiphany-platform/epiphany/issues/2873) - Epicli backup fails on schema validation
 - [#2886](https://github.com/epiphany-platform/epiphany/issues/2886) - [Ubuntu] [PostgreSQL] Apply command fails after upgrading from v1.0.x LTS to v1.3.0
+- [#2904](https://github.com/epiphany-platform/epiphany/issues/2904) - [BUG] Not possible to add PostgreSQL component to existing environment
```
Comment (to-bar) [inline docs/changelogs/CHANGELOG-1.3.md]:
```suggestion
- [#2904](https://github.com/epiphany-platform/epiphany/issues/2904) - Not possible to add PostgreSQL component to existing environment
```

**Review by to-bar — DISMISSED** [review]

**Comment by cicharka** [comment]
Conflict needs to be resolved!

**Review by seriva — DISMISSED** [review]

**Review by seriva — DISMISSED** [review]

**Review by to-bar — APPROVED** [review]

**Review by seriva — APPROVED** [review]

**Comment by przemyslavic** [comment]
Fixed. However, this will still block the installation in edge cases, such as re-applying after changing the number of single_machine nodes from 1 to 2, and in case of custom mappings for PostgreSQL:

apply:
```
    kubernetes_master:
      count: 1
    postgresql:
      count: 0  
```

```
---
kind: configuration/feature-mapping
title: Feature mapping to roles
name: default
provider: aws
specification:
  roles_mapping:
    kubernetes_master:
    - kubernetes-master
    - helm
    - applications
    - node-exporter
    - filebeat
    - postgresql
```

:x: re-apply:

```
    kubernetes_master:
      count: 1
    postgresql:
      count: 1
	  
```

```
---
kind: configuration/feature-mapping
title: Feature mapping to roles
name: default
provider: aws
specification:
  roles_mapping:
    kubernetes_master:
    - kubernetes-master
    - helm
    - applications
    - node-exporter
    - filebeat
    - postgresql
```

**Review by przemyslavic — APPROVED** [review]

**Pull request merged**